.circleci/config.yml

```diff
@@ -125,7 +125,8 @@ jobs:
       - run:
           name: '`pnpm dedupe` was run?'
           command: |
-            if [[ $(git diff --name-status next | grep pnpm-lock) == "" ]];
+            # #default-branch-switch
+            if [[ $(git diff --name-status master | grep pnpm-lock) == "" ]];
```

Comment:
Good thing that I tried these CI changes for prettier as it seems that this has been broken for a while. 🙈 
![Screenshot 2024-07-26 at 09 12 38](https://github.com/user-attachments/assets/8d3b59f5-42e2-4b53-b870-de843a79db27)
For reference: https://github.com/mui/material-ui/pull/41657

WDYT @Janpot, do we need all those cases like in the linked PR?
Are they there to catch cases where we forgot to run `install` and commit only a workspace and/or package change? 🤔

Reply:
> WDYT @Janpot, do we need all those cases like in the linked PR?

Probably not

> Are they there to catch cases where we forgot to run install and commit only a workspace and/or package change? 🤔

Yes 🙂

Reply:
Gotcha, let's try keeping it simple until we see that those cases are a problem. 👍